pkg/config/config.go

```diff
@@ -253,6 +255,10 @@ func GetConfig() *Config {
 	return config
 }
 
+func PasswordResetInterval() time.Duration {
```

Comment:
Travis build failed on the lint step. Exported functions (those with a name that starts by an upper case letter) must have a comment. Can you fix it?